src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsCard.component.ts

```diff
@@ -135,14 +159,36 @@ export class NotebookViewsCardComponent implements OnInit {
 	}
 
 	public get display(): boolean {
-		if (!this._metadata || !this._activeView) {
+		if (!this.activeView) {
```

Comment:
A lot of these properties could use comments - it's not clear to me without reading the code what they're supposed to mean. ? (seems like `visible` may be a better name for this anyways)

Reply:
Yeah, visible sounds like a good name. I added a few comments